src/App.js

```diff
@@ -559,23 +569,61 @@ class FeatureLoader extends React.Component<Props, State> {
     return { left: 50, right: 50, top: 50, bottom };
   }
 
+  renderFullscreenBackdrop() {
+    const routeInformation = getRouteInformation(this.props);
+    const { isEditMode } = routeInformation || {};
+    const isActive =
+      (this.state.isFilterToolbarVisible && this.state.isLocalizationLoaded) ||
+      this.state.isMainMenuOpen ||
+      this.state.isOnboardingVisible ||
+      this.state.isNotFoundVisible ||
+      isEditMode ||
+      this.state.isReportMode;
```

Comment:
0: Move to function?

Reply:
Yep! Will do that in the next PR, where I had to reorder some of these things.